action/protocol/staking/protocol.go

```diff
@@ -290,7 +294,17 @@ func (p *Protocol) CreatePreStates(ctx context.Context, sm protocol.StateManager
 			return err
 		}
 	}
-
+	if blkCtx.BlockHeight == p.config.FixAliasForNonStopHeight {
+		csm, err := NewCandidateStateManager(sm, featureWithHeightCtx.ReadStateFromDB(blkCtx.BlockHeight))
```

Comment:
Is this used to replace the original patch file, so the patch file is no longer needed?

Reply:
it will pro-actively create the patch, so the patch file is not needed later